engine-rest/engine-rest-openapi/src/main/templates/paths/authorization/{id}/options.ftl

```diff
@@ -0,0 +1,49 @@
+<#macro endpoint_macro docsUrl="">
+
+    <#-- NOTE: Any changes made to this file should be mirrored to the
+         ../options.ftl file as well -->
+{
+  <@lib.endpointInfo
+      id = "availableOperationsAuthorizationInstance"
+      tag = "Authorization"
+      summary = "Authorization Resource Options"
+      desc = "The `/authorization` resource supports two custom OPTIONS requests, one for the
+              resource as such and one for individual authorization instances. The
+              OPTIONS request allows you to check for the set of available
+              operations that the currently authenticated user can perform on the
```

Comment:
I would maybe reword the difference between this description and the one form the other options request.